sytest-blacklist

```diff
@@ -0,0 +1,17 @@
+# Blacklisted due to https://github.com/matrix-org/synapse/issues/1679
```

Comment:
a comment explaining what this file does would be useful.

also, blank lines between the entries? it's hard to read.